.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 1. Enhancement - Changed Liverpool (EGGP) SMR holding point labels from red to white - thanks to @zippy77777 (Darren Faux)
 2. Bug - Added Stansted (EGSS) and Cambridge (EGSC) SILVA1L STAR fix LOREL - thanks to @zippy77777 (Darren Faux)
 3. Bug - Changed Stapleford (EGSG) frequency from 122.800 to 122.825 - thanks to @bbrushfield (William Brushfield)
+4. AIRAC (2301) - Changed BCN to DME only
```

Comment:
```suggestion
4. AIRAC (2301) - Changed BCN to DME only - thanks to @Genau6502 
```
Thanks for the contribution @Genau6502 - see the suggestion above for the required part of the changelog entry. Additionally, you are welcome to leave your name in the format `- thanks to @Genau6502 (John Smith)`, or just commit as suggested above if you'd prefer to remain anonymous